parser/src/main/java/log/parser/controller/MatchController.java

```diff
@@ -0,0 +1,31 @@
+package log.parser.controller;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import log.parser.service.MatchService;
+
+@RestController
+@RequestMapping("api/match")
```

Comment:
When designing api's the convention is to use plural nouns for the resource urls

Reply:
Resource urls were updated